src/script/auth/page/Login.tsx

```diff
@@ -182,7 +182,8 @@ const LoginComponent = ({
       await doInitializeClient(ClientType.PERMANENT, undefined, undefined, entropyData);
 
       if (isOauth) {
-        return navigate(ROUTE.AUTHORIZE);
+        console.info('bardia route', {route: `${ROUTE.AUTHORIZE}/${window.location.hash}`});
```

Comment:
👀

Reply:
thanks for the catch: https://github.com/wireapp/wire-webapp/pull/15844/commits/158a9b9be132be161ba1c13e7b125d5224a9457e